.github/workflows/release.yaml

```diff
@@ -2,8 +2,11 @@ name: release
 
 on:
   push:
+    branches:
+      - main
+      - "v[0-9]+.[0-9]+"
     tags:
-      - 'v*'
+      - "v*"
```

Comment:
This will also publish release candidates. Is that OK or should we change it?

Reply:
I wondering what is the purpose of using `"v*"` instead of `'v*'` ?

yes, we are publicly publishing RC. to ease QA on public environment and also to allow external contributors/users to test release candidate.

Reply:
I don't know if simple vs double quotes make a difference in this particular case. I guess they're equivalent.